### PR TITLE
docs: fix loadbalancing-example

### PR DESCRIPTION
**Pull request by heikkilamarko**
Fixes to loadbalancing-example.

### CODE REVIEW

**Comment by hifabienne** [comment]
Hei @heikkilamarko Thanks for your pr
@eliobischof Can you please have a look at it?

File: docs/docs/self-hosting/deploy/loadbalancing-example/loadbalancing-example.mdx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export ZITADEL_MASTERKEY="$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)"
 docker compose up --detach
 ```
 
-Make `127.0.0.1` available at `my.domain`. For example, this can be achived with an entry `127.0.1.1 my.domain` in the `/etc/hosts` file.
+Make `127.0.0.1` available at `my.domain`. For example, this can be achived with an entry `127.0.0.1 my.domain` in the `/etc/hosts` file.
```
Comment (Lan-Phan) [inline docs/docs/self-hosting/deploy/loadbalancing-example/loadbalancing-example.mdx:65]: please help to correct typo here also
achived -> achieved

Comment (heikkilamarko) [reply docs/docs/self-hosting/deploy/loadbalancing-example/loadbalancing-example.mdx]: Typo fixed.

**Review by fforootd — APPROVED** [review]

**Comment by hifabienne** [comment]
hei @heikkilamarko 
Thanks for you contribution, we really appreciate it.
We would love to send you some swag from our side, if you are interested, please send an email to hi@zitadel.com

**Pull request merged**